homework/main.ipynb

```diff
@@ -727,131 +727,78 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyspark.sql import SparkSession\n",
+    "from pyspark.ml.feature import HashingTF, IDF, Tokenizer, MinHashLSH\n",
+    "from pyspark.sql.functions import col\n",
+    "import time\n",
+    "\n",
+    "# Initialize Spark session\n",
+    "spark = SparkSession.builder \\\n",
+    "    .appName(\"Barcelona_LSH\") \\\n",
+    "    .getOrCreate()\n",
+    "listings_df = spark.createDataFrame(listings_df)\n",
```

Comment:
Given class experience, I would say Diego wants us to read csv directly with Spark, not via pandas

Reply:
100%